.gitignore

```diff
@@ -8,6 +8,9 @@ build
 !.env.example
 vite.config.js.timestamp-*
 vite.config.ts.timestamp-*
+template_app/.svelte-kit
```

Comment:
This should be redundant with line 4, right?

Reply:
Ah yeah my bad

Reply:
Fixed in cda69ef